src/pages/auth/index.tsx

```diff
@@ -17,11 +19,21 @@ export default function Auth() {
     }));
   };
 
+  const handleSubmit = async (event: { preventDefault: () => void }) => {
+    event.preventDefault();
+    await axios.get("http://localhost/sanctum/csrf-cookie").then(() => {
```

Comment:
I think you can do ```await axios.get("http://localhost/sanctum/csrf-cookie")``` when the component mounts and do the axios.post in the handler